src/main/java/io/openmanufacturing/ame/resolver/inmemory/InMemoryStrategy.java

```diff
@@ -156,6 +157,8 @@ private List<Resource> getListOfAllBAMMElements( final KnownVersion version ) {
       resources.add( bamm.Constraint() );
       resources.add( bamm.AbstractEntity() );
       resources.add( bamm.AbstractProperty() );
+      resources.add( bamme.TimeSeriesEntity() );
```

Comment:
This should use `bamme.allEntities()` instead of referencing TimeSeriesEntity and ThreeDimensionalPosition explicitly.